src/app/customer/login/login.component.ts

```diff
@@ -2,33 +2,32 @@ import { Component } from '@angular/core';
 import { customer } from '../customer';
 import { CustomerService } from '../customer.service';
 import { Router } from '@angular/router';
+import { HttpErrorResponse } from '@angular/common/http';
 
 @Component({
   templateUrl: './login.component.html',
 })
 export class LoginComponent {
 
-  customer : customer = new customer();
+  customer: customer = new customer();
+  isLoginError: boolean = false;
 
-  constructor(private service1 : CustomerService, private router : Router){}
+  constructor(private service1: CustomerService, private router: Router) { }
```

Comment:
Why is the variable name service1??
Please follow proper naming conventions

Reply:
renamed service1 with customerService.